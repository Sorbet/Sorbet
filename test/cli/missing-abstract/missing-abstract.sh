#!/bin/bash
main/sorbet --silence-dev-message test/cli/missing-abstract/interface.rb test/cli/missing-abstract/implementation.rb test/cli/missing-abstract/reopen.rb 2>&1
main/sorbet --silence-dev-message test/cli/missing-abstract/implementation.rb test/cli/missing-abstract/reopen.rb test/cli/missing-abstract/interface.rb 2>&1
