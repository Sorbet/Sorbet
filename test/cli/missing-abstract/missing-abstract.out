test/cli/missing-abstract/implementation.rb:2: Missing definition for abstract method `Interface#foo` https://sorbet.org/docs/error-reference#5023
     2 |class Impl
        ^^^^^^^^^^
    test/cli/missing-abstract/interface.rb:8: defined here
     8 |  def foo
          ^^^^^^^
Errors: 1
test/cli/missing-abstract/implementation.rb:2: Missing definition for abstract method `Interface#foo` https://sorbet.org/docs/error-reference#5023
     2 |class Impl
        ^^^^^^^^^^
    test/cli/missing-abstract/interface.rb:8: defined here
     8 |  def foo
          ^^^^^^^
Errors: 1
